src/router/index.js

```diff
@@ -35,9 +36,25 @@ const routes = [
 			},
 			{
 				path: 'document/:document',
-				name: 'document',
-				component: Document,
-				meta: { title: 'Dokument' },
+				component: {
+					render(h) {
```

Comment:
Is this what you wanted? Didn't you want to simply nest `router-view`s? So you don't have to repeat some parts of the page which are same for `Document` and `DocumentCompare`.